docs/sources/mimir/configure/about-versioning.md

```diff
@@ -119,6 +119,9 @@ The following features are currently experimental:
 
 ## Deprecated features
 
+Deprecated features are usable until the release which indicates their removal.
```

Comment:
```suggestion
Deprecated features are usable until (and including) the release that indicates their removal.
```

Reply:
the release which indicates their removal does _not_ contain the features. So your edit wouldn't be correct.

Reply:
Please have another look at my edited feedback.